index.html

```diff
@@ -195,21 +227,26 @@ <h2>Skills</h2>
         </section>
 
         <section class="contact">
-            <h3>Contact me</h3>
-            <p>If you have an application you are interested in developing, a feature that you need built or a project
-                that needs coding. I’d love to help with it </p>
-            <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/moqzvegn"
-                method="post">
-                <!-- <label for="full-name">Full Name</label> -->
-                <input type="text" name="name" id="full-name" placeholder="Oussama Elabdioui" maxlength="30" required>
-                <!-- <label for="email-address">Email Address</label> -->
-                <input type="email" name="_replyto" id="email-address" placeholder="email@domain.tld" required><br>
-                <!-- <label for="message">Message</label> -->
-                <textarea rows="5" name="message" id="message" maxlength="500" placeholder="Write your message here"
-                    required></textarea>
-                <input type="hidden" name="_subject" id="email-subject" value="Contact Form Submission">
-                <input type="submit" id="submit" value="Get It Touch">
-            </form>
+            <div>
+                    <h3>Contact me</h3>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a
+                        project
+                        that needs coding. I’d love to help with it </p>
+                    <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8"
+                        action="https://formspree.io/f/moqzvegn" method="post">
+                        <!-- <label for="full-name">Full Name</label> -->
```

Comment:
- [ ] Please I think it will be more professional to remove all the commented code (lines 240 and 243 inclusive). ℹ️